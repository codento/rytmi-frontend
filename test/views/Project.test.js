import Vuex from 'vuex'
import { shallow, createLocalVue } from 'vue-test-utils'
import View from '../../src/views/Project.vue'

describe('Project.vue shallow', () => {
  let localVue = createLocalVue()
  localVue.use(Vuex)
  let store, getters, cmp

  const $route = {
    path: '/id',
    params: {id: 5}
  }
  beforeEach(() => {
    getters = {
      projectById: () => (arg) => arg
    }
    store = new Vuex.Store({ getters })

    cmp = shallow(View, {
      store,
      localVue,
      mocks: {
        $route
      }
    })
  })
  it('Template is correct', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
