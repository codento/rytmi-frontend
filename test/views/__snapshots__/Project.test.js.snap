// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project.vue shallow Template is correct 1`] = `
<div>
  <div
    class="animated fadeIn project-container col-sm-12 col-md-7"
  >
    <b-row>
      <b-col
        class="project-details"
      >
        <b>
          
        </b>
         
        <h1>
          
        </h1>
         
        <div
          class="detail-container"
        >
          <span
            class="detail detail-start"
          >
            <small>
              Start date
            </small>
            <br />
             
            <b>
              <!---->
            </b>
          </span>
           
          <span
            class="detail detail-end"
          >
            <small>
              End date
            </small>
            <br />
             
            <b>
              <!---->
            </b>
          </span>
           
          <span
            class="detail members"
          >
            <small>
              Consultants
            </small>
            <br />
             
            <!---->
          </span>
        </div>
         
        <p>
          <small>
            Description
          </small>
          <br />
          
          
        
        </p>
      </b-col>
    </b-row>
     
    <b-row>
      <b-col
        class="project-members"
      >
        <!---->
      </b-col>
    </b-row>
     
    <hr />
     
    <div>
      <h3
        class="project-profile-form-header"
      >
        Add a consultant
        
        <i
          class="fa fa-chevron-down"
        />
      </h3>
       
      <!---->
    </div>
  </div>
</div>
`;
